### ISSUE TITLE
The absence of the "kill all wine processes" feature

### ISSUE THREAD

**Issue by ztx-lyghters** [lutris/lutris#4266]
### Bug description

The absence of the "kill all wine processes" feature makes it's more difficult to use "Run EXE inside Wine prefix" option. The "game"/prefix is not considered "started" if you use "Run EXE inside WIne prefix" option and makes you RUN in in order to STOP it in order to KILL a not responding process.
Please, reconsider removing "kill" feature, you've just made it a bit harder for the user to handle some situations. One step is better than two, so this update is basically a downgrade in some sense.

### How to Reproduce

1. Go to the "bottom wine menu", whatever it is called
2. Click on "Run EXE inside WIne prefix"
3. Run something
4. Pretend it is not responding or somthing
5. Find the way to easily get rid of the wine server and all it's processes
6. Compare it to how you would solve this in previous Lutris versions

### Expected behavior

I expect Lutris to not lose features.

### Log output

```shell
-
```

### System Information

```shell
Doesn't matter.
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by Etaash-mathamsetty** [comment]
oh wait it's actually gone?
I thought lutris not detecting my terminal caused it

**Comment by strycore** [comment]
Yes, the feature was merged with the "Stop" button. The problem is that the Stop button only works when a game is launched. 
We could make the "Run EXE inside Wine prefix" feature compatible with the Stop button.

**Comment by kbulgrien** [comment]
I have observed situations where pressing the stop button (0.5.8.4) didn't work, and I had to ps -ef | grep -E (stuff|morestuff|etc) and kill processes.  Wasn't aware that running `wineboot -k` in the bottle was an option, but that makes sense.  Running it at the system level didn't do anything.  Unsure what version put this functionality into the "Stop" button was done, so perhaps my experience with Stop not working is irrelevant, but having to do the `ps` and kill thing is a real pain, and the whole thing about killing processes might have been more visible even in older versions.
